### ISSUE TITLE
Was 3.8.3-SNAPSHOT removed?

### ISSUE THREAD

**Issue by mboyd1993** [apollographql/apollo-kotlin#6121]
### Question

I've been using the 3.8.3-SNAPSHOT version of apollo-kotlin in a project and starting today that version can no longer be found.  I looked on sonatype.org to be sure and it looks like many of the other snapshot versions both older and newer than 3.8.3 were updated today, but 3.8.3-SNAPSHOT is gone.  Was that version intentionally removed for some reason?
https://s01.oss.sonatype.org/content/repositories/snapshots/com/apollographql/apollo3/apollo-gradle-plugin/

**Comment by BoD** [comment]
Hi! I'm not sure of how long snapshots are supposed to persist, but it's expected that they eventually disappear. Is there any reason why you're not using 3.8.3, or 3.8.5, or the latest 4.0.0?

**Comment by mboyd1993** [comment]
I had created a PR for a change to websocket behavior on iOS.  (#5704).  That PR got approved and @martinbonnin cherry-picked that change into version 3.8.3-SNAPSHOT for me to use pending the release of 4.0.0.  I'm currently in the process of updating to 4.0.0, but it's just taking me a bit to update gradle, update kotlin, etc and get the project to build again.  I hope to be done with that soon, and then we won't be using the SNAPSHOT version anymore, but in the meantime, my dev team is dead in the water because we need that change to the websocket behavior and 3.8.3-SNAPSHOT disappeared.   I thought it might be an oversight or something because earlier SNAPSHOT versions back to 3.0.1 got updated on Aug. 17 and 3.8.3 was the only one that disappeared.
